plugins/node/instrumentation-tedious/package.json

```diff
@@ -56,7 +56,8 @@
     "mocha": "7.2.0",
     "nyc": "15.1.0",
     "rimraf": "5.0.5",
-    "tedious": "15.1.3",
+    "semver": "^7.6.0",
+    "tedious": "^17.0.0",
```

Comment:
@pichlermarc Did you have strong preference (or particular reasons) for wanting pinned devDep versions? Now that we are using package-lock.json I'm curious if this pinning is an artifact of pre-package-lock days.

Reply:
@pichlermarc could you put your thoughts here? I also want to know if there is a reason for having pinned versions

Reply:
@trentm @david-luna it is a pre-package lock artifact. 

One benefit of keeping it pinned even after package-lock is added is that is that updates can be handled directly by renovate bot. That means arenovate PR will be scoped to that one package. Otherwise it would be lumped in together with lockfile updates. In such a lockfile PR it may become hard to pinpoint what's wrong.

Reply:
> That means arenovate PR will be scoped to that one package.

Sounds like a good reason to me. I've changed over to pinned versions in commit ddde198d.